packages/OptionButton/src/OptionButton.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import {
+  TextIcon,
+  OptionScaleIcon,
+  MultipleChoiceIcon,
+  LikertIcon,
+  TextIconDisabled,
+  OptionScaleIconDisabled,
+  MultipleChoiceIconDisabled,
+  LikertIconDisabled,
+} from './svgs';
+
+import './option-button.scss';
+
+export type ButtonType = 'Text' | 'OptionScale' | 'MultipleChoice' | 'Likert';
+
+export interface OptionButtonProps extends React.ComponentProps<'input'> {
+  /** True when the option button is selected */
+  checked?: boolean;
+  /** Add a specific class to the option button */
+  className?: string;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** Add a description that is displayed below the text prop */
+  description?: string;
+  /** True if the option button should be disabled */
+  disabled?: boolean;
+  /** Add an icon the the beginning of the option button.
+   * This overrides the icon that comes with buttonType */
+  icon?: React.ReactNode;
+  /** Add an id to the radio button of the option button.
+   * This will also be used as the htmlFor attribute on the label */
+  id: string;
```

Comment:
On `Toggle` and `Radio` we have named this props `htmlFor` to keep a constancy can you change it ?